wdtk-wikibaseapi/src/main/java/org/wikidata/wdtk/wikibaseapi/WbSearchEntitiesAction.java

```diff
@@ -69,7 +69,17 @@ public WbSearchEntitiesAction(ApiConnection connection, String siteUri) {
 	public List<WbSearchEntitiesResult> wbSearchEntities(WbGetEntitiesSearchData properties)
 			throws MediaWikiApiErrorException, IOException {
 		return wbSearchEntities(properties.search, properties.language,
-				properties.strictlanguage, properties.type, properties.limit, properties.offset);
+				properties.strictlanguage, properties.type, properties.limit, properties.offset, properties.uselang);
+	}
+
+	/**
+	 * Keeping this for backwards compatibility, the real action happens in 
+	 * {@link WbSearchEntitiesAction#wbSearchEntities(String, String, Boolean, String, Long, Long, String)}
+	 */
+	public List<WbSearchEntitiesResult> wbSearchEntities(String search, String language,
+														 Boolean strictLanguage, String type, Long limit, Long offset)
```

Comment:
Funny indentation here (and below too). I guess I need to get on with the linting setup (we have something that works quite well in OpenRefine).

Reply:
Yeah, the intelliJ autoformat and the format used here do not get along well